config.json

```diff
@@ -78,6 +87,10 @@
         "width": 300,
         "height": 300
       },
+      "productGallery": {
```

Comment:
because `gallery` applies to more than just products, but since we are currently allowing users to change product gallery size, but we made a new one just for products.